# The EMP500 Dataset

The Earth Microbiome Project 500 (EMP500) was a multi-omic study of microbial
communities sampled from various biomes and habitats of the planet. The study
was published in:

- Shaffer, J. P., Nothias, L. F., Thompson, L. R., Sanders, J. G., Salido, R.
  A., Couvillion, S. P., ... & Knight, R. (2022). Standardized multi-omics of
  Earthâ€™s microbiomes reveals microbial and metabolite diversity. Nature
  Microbiology, 7(12), 2128-2150.
- https://www.nature.com/articles/s41564-022-01266-x

The raw and processed data of EMP500 are available from several public sources,
including:

- https://earthmicrobiome.org/emp500/
- https://github.com/biocore/emp/
- Qiita (study ID: 13114)
- ENA (project: PRJEB42019)
- MassIVE (accession: MSV000083475)
- GNPS

This repository hosts an organized collection of processed data generated from
the EMP500 project. Descriptions of directories and files are provided below.


## Sample metadata

`sample.tsv`: Sample metadata table

- 880 samples, 33 fields


## Amplicon sequencing

`amplicon/`: Amplicon sequencing data

16S rRNA V4 amplicon sequencing was performed on an Illumina platform.
Sequencing reads were processed on the Qiita platform using the Deblur
workflow to generate 150 bp amplicon sequence variants (ASVs). Results
were mapped against the Greengene 2 (GG2) database (release 2022.10).

- `16s.biom`: Feature table of ASVs (GG2 sequence IDs).

  - 805 samples, 427,269 features (ASVs)

- `16s.fa`: ASV sequences corresponding to IDs.
- `16s.nwk`: GG2 reference phylogeny pruned to the current features.
- `16s.tax`: GG2 reference taxonomy filtered to the current features.


## Shotgun metagenomic sequencing

`shotgun/`: Shotgun metagenomic sequencing data

Whole metagenome shotgun sequencing was performed on an Illumina platform to
generate 150 bp paired-end reads. Sequencing data were processed on the Qiita
platform. Reads were mapped against the Web of Life 2 (WoL2) database.

- `ogu.biom`: Feature table of operational genomic units (OGUs). Each OGU
  represents a reference microbial genome. Cell values represent the number
  of sequencing reads mapped to each OGU.

  - 801 samples, 14,933 features (OGUs)

- `ogu.nwk`: WoL2 reference phylogeny.
- `ogu.tax`: WoL2 reference taxonomy.
- `ogu.info`: Information of reference genomes.

- `ko.biom`: Feature table of KEGG orthologs (KOs). Cell values represent the
  number of sequencing reads mapped to genes assigned to each KO.

  - 799 samples, 9,319 features (KOs)

- `ko.name`: KO names, including gene name, product name and EC number.

- `ec.biom`: Feature table of Enzyme Commission (EC) numbers. Generated by
  collapsing KOs according to the KEGG system.

  - 797 samples, 3,326 features (EC numbers)

- `pathway.biom`: Feature table of KEGG pathways. Generated by collapsing KOs
   according to the KEGG system.

  - 787 samples, 110 features (KEGG pathways)

- `pathway.name`: Pathway names.


## Metagenomic assembly

Shotgun metagenomic sequencing data were co-assembled by environment and binned
into metagenome-assembled genomes (MAGs). Only high-quality MAGs (n = 811) were
selected.

`assembly/`: Metagenome-assembled genomes (MAGs)

- `mag.biom`: Feature table of MAG abundances per sample. Generated by mapping
  sequencing reads against MAG sequences using Bowtie2.

  - 813 samples, 811 features (MAGs)

- `fasta/`: Sequences of assembly scaffolds within each MAG.
- `mag.tax`: Taxonomic assignments of MAGs. Classified using GTDB-Tk 2.4.0
  against GTDB R220.
- `prokka`: Annotation results of MAGs. Generated using Prokka v1.14.6.
  - `faa/`: Protein sequences of predicted open reading frames (ORFs)
  - `ffn/`: Coding (DNA) sequences of the ORFs.
  - `gff/`: Information of genes and other features predicted from the MAG
    sequences.


## Untargeted metabolomics

`masspec/`: Mass spectrometry data

Untargeted metabolomics was performed using liquid chromatography-tandem mass
spectrometry (LC-MS/MS). Spectrum data were processed using MZmine and analyzed
on the GNPS platform.

- `lcms.biom`: Feature table of spectra (i.e., metabolites). Cell values
   represent ion intensity (peak area) of each spectrum.

  - 618 samples, 52,496 features (spectra)

- `fbmn.tsv`: Annotation of spectra. Generated using the feature-based
  molecular networking workflow (FBMN) on GNPS.
- `canopus.tsv`: Classification of spectra according to the NPClassifier
  ontology (formula, class, superclass, pathway). Predicted by CANOPUS.
- `is.microbial`: Whether each spectrum represents a microbially related
  secondary metabolite (0 - no, 1 - yes), based on the annotation results by
  multiple tools.

